content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -170,6 +170,7 @@ Unable to connect to the server: x509: certificate signed by unknown authority (
 
   ```sh
   mv  $HOME/.kube $HOME/.kube.bak
+  mkdir $HOME/.kube
```

Comment:
i think this should use `cp -r` to copy the directory to a backup `.bak`.
then `sudo cp -i....` the `admin.conf`.
what do you think?

Reply:
It seems like the 2 separate use-cases need to be called out in the script. Ex:

```
# If you have a `$HOME/.kube`(check with: ls $HOME/ | grep kube)
# If you don't have it
create it with `mkdir`
... or something
```

Reply:
IMO, if change to `mkdir -p $HOME/.kube`, LGTM